src/schema/v2/homeView/zones/default.ts

```diff
@@ -1,33 +1,35 @@
+import { AuctionLotsForYou } from "../sections/AuctionLotsForYou"
 import { Auctions } from "../sections/Auctions"
+import { CuratorsPicksEmerging } from "../sections/CuratorsPicksEmerging"
+import { DiscoverMarketingCollections } from "../sections/DiscoverMarketingCollections"
+import { DiscoverSomethingNew } from "../sections/DiscoverSomethingNew"
+import { ExploreByCategory } from "../sections/ExploreByCategory"
+import { ExploreByMarketingCollectionCategories } from "../sections/ExploreByMarketingCollectionCategories"
+import { FeaturedFairs } from "../sections/FeaturedFairs"
+import { GalleriesNearYou } from "../sections/GalleriesNearYou"
+import { HeroUnits } from "../sections/HeroUnits"
+import { LatestActivity } from "../sections/LatestActivity"
 import { LatestArticles } from "../sections/LatestArticles"
-import { News } from "../sections/News"
 import { LatestAuctionResults } from "../sections/LatestAuctionResults"
-import { LatestActivity } from "../sections/LatestActivity"
-import { ViewingRooms } from "../sections/ViewingRooms"
-import { ShowsForYou } from "../sections/ShowsForYou"
 import { MarketingCollections } from "../sections/MarketingCollections"
-import { FeaturedFairs } from "../sections/FeaturedFairs"
-import { HeroUnits } from "../sections/HeroUnits"
-import { RecommendedArtists } from "../sections/RecommendedArtists"
-import { TrendingArtists } from "../sections/TrendingArtists"
-import { ActiveBids } from "../sections/ActiveBids"
-import { RecommendedArtworks } from "../sections/RecommendedArtworks"
-import { NewWorksFromGalleriesYouFollow } from "../sections/NewWorksFromGalleriesYouFollow"
+import { News } from "../sections/News"
 import { NewWorksForYou } from "../sections/NewWorksForYou"
-import { AuctionLotsForYou } from "../sections/AuctionLotsForYou"
+import { NewWorksFromGalleriesYouFollow } from "../sections/NewWorksFromGalleriesYouFollow"
 import { RecentlyViewedArtworks } from "../sections/RecentlyViewedArtworks"
-import { CuratorsPicksEmerging } from "../sections/CuratorsPicksEmerging"
+import { RecommendedArtists } from "../sections/RecommendedArtists"
+import { RecommendedArtworks } from "../sections/RecommendedArtworks"
+import { ShowsForYou } from "../sections/ShowsForYou"
 import { SimilarToRecentlyViewedArtworks } from "../sections/SimilarToRecentlyViewedArtworks"
-import { isSectionDisplayable } from "../helpers/isSectionDisplayable"
-import { DiscoverMarketingCollections } from "../sections/DiscoverMarketingCollections"
-import { ExploreByMarketingCollectionCategories } from "../sections/ExploreByMarketingCollectionCategories"
-import { ExploreByCategory } from "../sections/ExploreByCategory"
-import { DiscoverSomethingNew } from "../sections/DiscoverSomethingNew"
+import { Tasks } from "../sections/Tasks"
+import { TrendingArtists } from "../sections/TrendingArtists"
+import { ViewingRooms } from "../sections/ViewingRooms"
 
 const SECTIONS: HomeViewSection[] = [
+  Tasks,
```

Comment:
do we want to keep this along with the latest activity in the future? should this be also controlled by a ff

Reply:
Feature flag is controlled [here](https://github.com/artsy/metaphysics/pull/6095/files#diff-b76fbec1d39f9d817430b50200a3bb5b3068ad43f07fdfedff386d67a8f6fe82R16) as part of the section instance definition. This section won't show up unless the authenticate user is eligible via the Unleash feature flag configuration.

Reply:
> do we want to keep this along with the latest activity in the future?

@MounirDhahri for now, I guess we will display both next to each other on the home screen. It's a bit weird because they seem very similar, but it should be okay because the "Act Now"/Tasks section is only displayed if the user has any open tasks.